tests/test_lightningd.py

```diff
@@ -110,45 +110,45 @@ def tearDown(self):
 
 
 class LightningDTests(BaseLightningDTests):
-    def test_connect(self):
+    def connect(self):
```

Comment:
I think I'll generalize this a bit further to create arbitrary length line-graphs, since that's what we mostly test.